bootloader/bin/bootstrap.sh

```diff
@@ -129,21 +129,23 @@ print_selector() {
   echo "│ Shimboot OS Selector │"
   echo "└──────────────────────┘"
 
-  if [ "${rootfs_partitions}" ]; then
+  # Check if there are bootable partitions
+  if [ -n "${rootfs_partitions}" ]; then
+    # Process each rootfs partition entry
     for rootfs_partition in $rootfs_partitions; do
-      #i don't know of a better way to split a string in the busybox shell
-      local part_path=$(echo $rootfs_partition | cut -d ":" -f 1)
-      local part_name=$(echo $rootfs_partition | cut -d ":" -f 2)
+      # Use IFS to split the string, separating by `:`
+      IFS=':' read -r part_path part_name _ <<< "$rootfs_partition"
```

Comment:
I honestly think that the method with `cut` is more readable and easier to understand.

Reply:
Yeah but again this minimizes use of external commands